src/daos_vol_public.h

```diff
@@ -59,20 +59,33 @@ extern "C" {
 /**
  * Modify the file access property list to use the DAOS VOL connector.
  *
- * \pool_uuid identifies the UUID of the DAOS pool to connect to.
+ * \pool identifies the UUID or label of the DAOS pool to connect to.
  *
- * \pool_grp identifies the server group name to use when connecting to
- * the DAOS pool. This may be NULL, in which case a default group name
+ * \sys_name identifies the DAOS system name to use when connecting to
+ * the DAOS pool. This may be NULL, in which case a default system name
  * is used.
  *
  * \param fapl_id   [IN]    File access property list
- * \param pool_uuid [IN]    DAOS pool UUID
- * \param pool_grp  [IN]    Process set name of the DAOS servers managing the pool
+ * \param pool      [IN]    DAOS pool UUID
+ * \param sys_name  [IN]    Process set name of the DAOS servers managing the pool
  *
  * \return Non-negative on success/Negative on failure
  */
 H5VL_DAOS_PUBLIC herr_t
-H5Pset_fapl_daos(hid_t fapl_id, const uuid_t pool_uuid, const char *pool_grp);
+H5Pset_fapl_daos(hid_t fapl_id, const char *pool, const char *sys_name);
```

Comment:
I think the example programs will probably need to be updated after this API change. There are a few references in the connector's documentation as well that could use updating so they aren't out of sync.

Reply:
ah yes i didn't have build examples in cmake options, ill add that and fix those